Dockerfile

```diff
@@ -0,0 +1,32 @@
+ARG commit=develop
+FROM elifesciences/peerscout_client:${commit} AS client
+FROM elifesciences/python:cacd250ec201feab491c0738f261561b5360997b
+
+USER elife
+ENV PROJECT_FOLDER=/srv/peerscout
+RUN mkdir ${PROJECT_FOLDER}
+WORKDIR ${PROJECT_FOLDER}
+COPY --chown=elife:elife install.sh requirements.txt requirements-debug.txt ${PROJECT_FOLDER}/
+RUN /bin/bash install.sh
+
+COPY --from=client --chown=elife:elife /home/node/client/ ${PROJECT_FOLDER}/client/
+COPY --chown=elife:elife docvec_model/ ${PROJECT_FOLDER}/docvec_model/
```

Comment:
`docvec_model` was used to support unpickling models that were pickled before everything was moved into the `peerscout` package. It probably can be removed now.